clang/lib/Frontend/VerifyDiagnosticConsumer.cpp

```diff
@@ -1098,7 +1098,16 @@ void VerifyDiagnosticConsumer::CheckDiagnostics() {
     // Produce an error if no expected-* directives could be found in the
     // source file(s) processed.
     if (Status == HasNoDirectives) {
-      Diags.Report(diag::err_verify_no_directives).setForceEmit();
+      std::string directives;
+      for (auto &Prefix : Diags.getDiagnosticOptions().VerifyPrefixes) {
+        directives = directives + Prefix + ",";
+      }
```

Comment:
In `PrintUnexpected` we're simply using `Diags.getDiagnosticOptions().VerifyPrefixed.begin`.  How is this case different?

Reply:
yes suppose multiple diagnostics were not found like given in the third case here : 

> ![Screenshot from 2024-01-20 00-23-12](https://private-user-images.githubusercontent.com/114918019/298168762-c8c7dcca-a1eb-4ee5-8fe7-53c7c2ac99ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5NDU2NTIsIm5iZiI6MTcwNTk0NTM1MiwicGF0aCI6Ii8xMTQ5MTgwMTkvMjk4MTY4NzYyLWM4YzdkY2NhLWExZWItNGVlNS04ZmU3LTUzYzdjMmFjOTljYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyMlQxNzQyMzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZTRjNmIwYmU4MDhhMGNhZjc3NzE2NTgwYmUzNjRiZjEzNDE5YzhmNTJmYmNhZDM3ZDBmZTg5NTUwNGViNzc2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.CEGvxZlT56vu6qZ73y9ZSxX4d2a2FhH5hYEW3Wp6sUM)

then in the error generated, I give all the diagnostics as ```consider use of 'foo,alpha,bar,gamma-no-diagnostics'```. But if that is not the intended behavior or if multiple errors are expected to be emitted in response to multiple diagnostics not found, then that can also be done. Please specify which of the two is required.

Reply:
I think the one I implemented is a better option as it batched the same types of errors into one and gave the programmer a concise view of the error.

Reply:
Well but I can't add a "foo,alpha,bar,gamma-no-diagnostics"(literally) comment to the test, right? That doesn't work?

Reply:
Yes, I see your point. In that case I will change the code to now emit multiple errors and not batch them into one. so if you pass ```-verify=foo,bar``` and there are no such directives in the code then two errors with ```foo-no-diagnostics``` and ```bar-no-diagnostics``` will be generated. Please give me a confirmation to do so.

Reply:
Considering this as resolved.